sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -40,14 +40,15 @@ internal open class KlaviyoApiRequest(
         const val COMPANY_ID = "company_id"
         const val PROFILE = "profile"
         const val EVENT = "event"
+        const val PUSH_TOKEN = "push-token"
 
         // Headers
         const val HEADER_CONTENT = "Content-Type"
         const val HEADER_USER_AGENT = "User-Agent"
         const val HEADER_ACCEPT = "Accept"
         const val HEADER_REVISION = "Revision"
         const val TYPE_JSON = "application/json"
-        const val V3_REVISION = "2023-01-24"
+        const val V3_REVISION = "2023-06-06.pre"
```

Comment:
feels weird to be using a `pre` revision of the api

Reply:
💯, can't release this till the tokens API is public.